index.html

```diff
@@ -0,0 +1,68 @@
+                            <div id="menu-outer">
+                            <div class="table">
+                                <ul id="horizontal-list">
+                                    <li><a href="#"><div></div></a></li>
+                                    <li><a href="#">Store</a></li>
+                                    <li><a href="#">Mac</a></li>
+                                    <li><a href="#">Ipod</a></li>
+                                    <li><a href="#">Iphone</a></li>
+                                    <li><a href="#">Iped</a></li>
+                                    <li><a href="#">Itunes</a></li>
+                                    <li><a href="#">Support</a></li>
+                                    <li><input type="text"></li>
+                                </ul>
+                            </div>
+                        </div>
+                        
+                    </header>
+                    <div class="text">
+                        <img class="text1" src="img/text1.png" width="980" height="100" alt="What will your verse be?">
+                        <img class="text2" src="img/text2.png" width="980" height="82" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+                        <img class="text3" src="img/text3-b.png" width="980" height="32" alt="Watch the “Your Verse” film and explore the stories">
+                    </div>
+        </main>
+        <div class="contentN">
+            <div class="conUl">
+                <a href="#"><div></div></a>
+                <a href="#"><div></div></a>
+                <a href="#"><div></div></a>
+                <a href="#"><div></div></a>
+            </div>
+        </div>
+        <footer class="footStyle">
+                <div class="footContainer">
+                        <div class="div-left">
+                            <p class="">
+                                Shop the <a href="#">Apple Online Store</a>(1-800-MY-APPLE), visit an <a href="#">Apple Retail Store</a>, or find a <a href="#">reseller</a>.
+                            </p>
+                        </div>
+                        <div class="div-right">
+                            <ul class="right">
+                                <li><a href="#">Site Map</a></li>
+                                <li><a href="#">Hot News</a></li>
+                                <li><a href="#">RSS Feeds</a></li>
+                                <li><a href="#">Media Info</a></li>
+                                <li><a href="#">Environment</a></li>
+                                <li><a href="#">Job Opportunites</a></li>
+                                <li><a href="#">Contact Us</a></li>
+                                <li><a href="#"><img alt="Flag not found" src="img/usa.png"></a></li>
+                            </ul>
+                        </div>
+                        <div class="clear left">
```

Comment:
In this `<div>` tag, in particular, the `clear` class appended to this tag is giving the entire container a `padding-left: 6px` property which is also one of the reasons it does not look aligned.

Reply:
xD thank you so much for your explanation i fixed everything as you said.
<p> tag text-align changed to left, Class .clear padding left removed :)

Reply:
@jofer86 xD thank you so much for your clean explanation i fixed everything as you said.
<p> tag text-align changed to left, Class .clear padding left removed :)